packages/webpack/mixins/config/mixin.core.js

```diff
@@ -56,6 +56,18 @@ class WebpackConfigMixin extends Mixin {
     }
     if (target === 'develop' || target === 'build') {
       configLoaderConfig.options.type = 'browser';
+      configLoaderConfig.options.config = {
+        ...this.config,
+        _config: Object.entries(this.config.browserWhitelist).reduce(
```

Comment:
This only takes care of the config that is being bundled by webpack.
Another thing to watch out for would be the `_env` variables, which will still be visible in the generated HTML.
If we want to remove the corresponding `_env` entries too, we are probably going to mix client/server concerns or mix the implementation between `@untool/webpack` and `@untool/react`..